src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -662,4 +666,19 @@ public ResponseEntity<Announcements> getAnnouncements() {
                 .ok()
                 .body(announcementsService.getAnnouncements());
     }
-}
+
+    /**
+     * Update Ootb active user profile.
+     */
+    @PostMapping(value = "/activeProfile/ootb")
+    public ResponseEntity<Boolean> updateOotbActiveUserProfile(@RequestBody List<String> authorities,
```

Comment:
Instead of returning a Boolean in the response, we tend to give back all data passed in with a SUCCESS result code. Something like this:
```
{
    "resultCode": "SUCCESS",
    "result": {
        "uid": "",
        "uhUuid": "",
        "name": "",
        "givenName": "",
        "authorities": []
    }
}
```